unyt/tests/test_unyt_array.py

```diff
@@ -2379,3 +2379,29 @@ def test_neper_bel():
         unyt_array([0, 20], "dB") ** 2
     with pytest.raises(InvalidUnitOperation):
         np.power(unyt_array([0, 20], "dB"), -2)
+
+
+def test_delta_degC():
+    t1 = 10 * degC
+    t2 = 1 * K
+    assert t1 + t2 == 11 * degC
+    with pytest.warns(FutureWarning):
+        t2 + t1
+    t3 = 1 * delta_degC
+    assert t1 + t3 == 11 * degC
+    assert t3 + t1 == 11 * degC
+    assert 1 * delta_degC + 2 * delta_degC == 3 * delta_degC
+    assert 2 * delta_degC == unyt_quantity(2, "delta_degC")
+
+
+def test_delta_degF():
+    t1 = 10 * degF
+    t2 = 1 * R
+    assert t1 + t2 == 11 * degF
+    with pytest.warns(FutureWarning):
+        t2 + t1
```

Comment:
Same here, we should check that the result is still what we expect.

Reply:
Added assertion